wifiphisher/macmatcher.py

```diff
@@ -0,0 +1,45 @@
+import os
+from constants import *
+
+class MACMatcher():
+    """
+        This class is using Organizationally Unique Identifiers (OUIs) 
+        in order to match MAC addresses of devices to its vendord  
+
+        See http://standards.ieee.org/faqs/OUI.html
+    """
+    def __init__(self, mac_prefix_file=MAC_PREFIX_FILE):
+        self.mac_to_vendor = {}
+
+        path = os.path.join(os.path.dirname(__file__), '..', mac_prefix_file)
```

Comment:
`mac_prefix_file` needs to be inside the `wifiphisher` directory. Then you won't worry about joining any paths.